doc/introduction/importing_circuits.rst

```diff
@@ -0,0 +1,272 @@
+.. role:: html(raw)
+   :format: html
+
+.. _intro_ref_importing_circuits:
+
+Importing circuits
+==================
+
+PennyLane offers the :mod:`~.io` module to import quantum circuits and operations that were
+constructed outside of PennyLane. This includes circuits defined using `Qiskit <https://www.ibm.com/quantum/qiskit>`__,
+`OpenQASM <https://docs.quantum.ibm.com/build/interoperate-qiskit-qasm2>`_, and `Quil
```

Comment:
```suggestion
`OpenQASM <https://arxiv.org/abs/1707.03429>`_, and `Quil
```
Unfortunately the main openQasm docs are for v3, so maybe we just link to the original paper

Reply:
Good idea! Thanks for finding the link. 🔗